src/networkextension.cs

```diff
@@ -327,6 +327,11 @@ interface NEDnsSettings : NSSecureCoding, NSCopying
 		[NoWatch, NoTV, Mac (11, 0), iOS (14, 0)]
 		[Export ("dnsProtocol")]
 		NEDnsProtocol DnsProtocol { get; }
+
+		[Mac (11,0), iOS (14,0), NoTV]
```

Comment:
it should either be `[NoWatch]` (like above) or `[Watch (7,0)]`
unless there's defines (that I don't see in the diff) the current attributes makes it available on watchOS 2.0

Reply:
NetworkExtension is not present on the watch